quack_app/test/core/notification_data_test.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:quack_app/core/notification_data.dart';
+
+void main() {
+  group("MenuData", () {
+    NotificationData notificationData = NotificationData();
+    test("getExistingNotifications()", () async {
+      await notificationData.getExistingNotifications();
+      expect(notificationData.currentNotifications.length, 3);
+    });
+
+    test("remove()", () async {
+      NotificationObject first = notificationData.currentNotifications[0];
+      notificationData.remove(first.id);
+      expect(notificationData.currentNotifications.length, 2);
```

Comment:
Same here - rather than hard coding the length, get the length of total notifications and expect `length - 1` rather than hard coding `2`